backend/src/Squidex.Web/Resources.cs

```diff
@@ -20,17 +20,19 @@ public sealed class Resources
         private readonly Dictionary<(string, string), bool> schemaPermissions = new Dictionary<(string, string), bool>();
 
         // Contents
-        public bool CanReadContent(string schema) => IsAllowedForSchema(P.AppContentsRead, schema);
+        public bool CanReadContentOwn(string schema) => IsAllowedForSchema(P.AppContentsRead, schema);
```

Comment:
Gain, please do not change the names here, only the permissions that are used.

it should be:

```
public bool CanReadContent(string schema) => IsAllowedForSchema(P.AppContentsRead**Own**, schema);
```